Bindings/DebugBinding.cs

```diff
@@ -1,17 +1,28 @@
-﻿using System.Threading;
-using TechTalk.SpecFlow;
-
-namespace Behavioral.Automation.Bindings
-{
-    [Binding]
-    public sealed class DebugBinding
-    {
-        [Given("wait")]
-        [When("wait")]
-        [Then("wait")]
-        public void Wait()
-        {
-            Thread.Sleep(5000);
-        }
-    }
-}
+﻿using System;
+using System.Threading;
+using TechTalk.SpecFlow;
+using JetBrains.Annotations;
+
+namespace Behavioral.Automation.Bindings
+{
+    [Binding]
+    public sealed class DebugBinding
+    {
+        [Given("wait")]
+        [When("wait")]
+        [Then("wait")]
+        public void Wait()
+        {
+            Thread.Sleep(5000);
+        }
+
+        [Given("wait (.*) sec")]
+        [When("wait (.*) sec")]
+        [Then("wait (.*) sec")]
+        public void Wait([NotNull] string sec)

```

Comment:
as far as I remember, you can avoid usage of string and use int here. Auto conversion should be made.

Reply:
Done